cmd/theatre-consoles/main.go

```diff
@@ -184,10 +185,17 @@ func LifecyclePrinter(logger kitlog.Logger) runner.LifecycleHook {
 			)
 			return nil
 		},
-		ConsoleRequiresAuthorisationFunc: func(csl *workloadsv1alpha1.Console) error {
+		ConsoleRequiresAuthorisationFunc: func(csl *workloadsv1alpha1.Console, rule *workloadsv1alpha1.ConsoleAuthorisationRule) error {
+			authoriserSlice := make([]string, 0, len(rule.ConsoleAuthorisers.Subjects))
+			for _, authoriser := range rule.ConsoleAuthorisers.Subjects {
+				authoriserSlice = append(authoriserSlice, authoriser.Kind+":"+authoriser.Name)
+			}
+			authorisers := strings.Join(authoriserSlice, ",")
```

Comment:
Just a nitpick, should we add a space after the comma?

Reply:
This was a concious decision. When printing it will appear in logfmt. As this is space separated joining on `", "` wouldn't work very well.